### ISSUE TITLE
you releasing obfuscated code on github?

### ISSUE THREAD

**Issue by wendellmeset** [SmileMan1235/SkidHub#2]
lol, just plain, lol

**Comment by Tacosheel** [comment]
@wendellmeset big retard moment

**Comment by wendellmeset** [comment]
@Tacosheel nah, `"return(function(e,...)local j="This file was obfuscated using PSU Obfuscator 4.0.A | https://www.psu.dev/ & discord.gg/psu";local C=e"`
